system/vmware-monitoring/Chart.yaml

```
@@ -1,5 +1,6 @@
 apiVersion: v2
 name: vmware-monitoring
+=======
 version: 2.1.3
```

bump is still needed and the additional line needs to go.
```suggestion
version: 2.1.4
```